docs/functions/Bundles/Start-UPloadVCFBundle.md

```diff
@@ -1,38 +0,0 @@
-# Start-UploadVCFBundle
-
-### Synopsis
-The Start-UploadVCFBundle starts upload of bundle to SDDC Manager
-
-### Syntax
-```
-Start-UploadVCFBundle -json <path to json file>
-```
-
-### Description
-The Start-UploadVCFBundle cmdlet starts upload of bundle(s) to SDDC Manager
```

Comment:
Is this from a local file system ?

Reply:
Module Name should be 'Start-VCFUploadBundle'

Reply:
same as above. It's a deleted file